frontend/src/lib/components/Description/Description.tsx

```diff
@@ -0,0 +1,60 @@
+import { Card, Input } from 'antd'
+import React, { Ref, useState } from 'react'
```

Comment:
This is a component particularly scoped for insights / dashboards, not sure if this is the best location to save it (may lead to confusion). Also, adding a short description on what this component does might be helpful.